ansible/roles/configure/files/helpers/swssconfig_args_update.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
This script is lacking comment. Could you also give some samples?
More general question: is it ever possible to test the DUT without changing the content of the docker container?